idunn/utils/result_filter.py

```diff
@@ -340,7 +340,8 @@ def build_params_from_bragi(cls, query: str, bragi_response: dict) -> dict:
         local_names = [
             prop["value"]
             for prop in bragi_response.get("properties", [])
-            if prop["key"].startswith("name:") or prop["key"].startswith("alt_name:")
+            if isinstance(prop, dict)
```

Comment:
When is this happening? The field may be `null`?

Reply:
This is happening on `properties` of Tripadvisor POI 
```
  "properties": {
    "ta:average_rating": "4.5",
    "ta:photos_url": "https://www.tripadvisor.com/Restaurant_Review-g1743691-d3166925-Reviews-Chez_Eric-Vaucluse_Provence_Alpes_Cote_d_Azur.html?m=66562#photos",
    "ta:review_count": "321",
    "ta:url": "https://www.tripadvisor.com/Restaurant_Review-g1743691-d3166925-Reviews-Chez_Eric-Vaucluse_Provence_Alpes_Cote_d_Azur.html?m=66562",
    "website": "http://www.facebook.com/profile.php?id=778236518876620",
    "image": "https://media-cdn.tripadvisor.com/media/photo-o/0f/e9/04/82/photo0jpg.jpg"
  }
```
is different from OSM POI : 
```
  "properties": [
    {
      "key": "addr:street",
      "value": "Rue de la Légion d'Honneur"
    }
 ]
```